app/views/staff/report/_combined_priorities.html.haml

```diff
@@ -16,6 +16,7 @@
         Percentage completed on time
   %tbody.govuk-table__body
     - presenter.priorities.each do |priority|
+      - next if presenter.defects_by_priority(priority: priority).count < 1
```

Comment:
I'm not a big fan of too much logic in templates. Can we add a `priorities_with_defects` method or similar to the presenter here?